spanner/api/Spanner.Samples/ListInstanceConfigOperations.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_list_instance_config_operations]
+
+using System;
+using System.Collections.Generic;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.LongRunning;
+
+namespace Spanner.Samples
+{
+    public class ListInstanceConfigOperationsSample
+    {
+        public IEnumerable<Operation> ListInstanceConfigOperations(string projectId)
+        {
+            var instanceAdminClient = InstanceAdminClient.Create();
+            var projectName = ProjectName.FromProject(projectId);
+            var listInstanceConfigOperations = instanceAdminClient.ListInstanceConfigOperations(projectName);
+
+            foreach (var operation in listInstanceConfigOperations)
```

Comment:
Please only print 5 elements, the same as in other listing examples, to make sure that we are not hitting the server several times, we were reaching admin quota very easily.

See this commit: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/commit/35baed2aa2a6491523cebfa9d301750088af13b0